prbt_hardware_support/README.md

```diff
@@ -3,33 +3,38 @@ The prbt_hardware_support package contains files supporting the certification of
 
-## SBC
-The SBC function ("Safe brake control") of the robot arm is a safety function which is used in conjunction with the STO and prevents a motion when the torque of the drives is turned off.
+| Parameter Description| Options | Argument name in <br> `prbt_support/robot.launch`| Option values in <br> `prbt_support/robot.launch`
+| - | - | - | - |
+| Gripper Model | *No gripper*, Schunk PG plus 70 | `gripper` | *`<empty_string>`*, `pg70` |
+| Safety Controller Hardware         | _PSS4000_, PNOZmulti | `safety_hw` | *`pss4000`*, `pnoz` |
+| Brake Test Support | *True*, False | `has_braketest_support` | *`true`*, `false` |
+| Operation Mode Support| *True*, False | `has_operation_mode_support` | *`true`*, `false` |
+| Visual Status Indicator| *True*, False | `visual_status_indicator` | *`true`*, `false` |
 
-# Safe stop 1 (SS1)
-To allow a controlled stop, the safety controller delays the STO signal by several milliseconds. This package opens a
-modbus connection to the safety controller (PNOZmulti or PSS4000). The safety controller sends an emergency
-stop signal via Modbus immediately so that ros_control has a short time interval to stop the drives via a brake ramp.
-The TCP could for example brake on the current trajectory. After execution of the brake ramp, the drivers are halted. Even if ROS would fail, the safety controller turns off the motors via STO (that would be a Stop 0 then).
+For more on gripper models see also [prbt_grippers](https://github.com/PilzDE/prbt_grippers).
 
-## Possible error cases and their handling
+# Safety Features
 
-| Error cases                                             | Handling                                                |
-| ------------------------------------------------------- | ------------------------------------------------------- |
-| Modbus client crashes                                   | ROS system is shutdown which leads to an abrupt stop of the robot. |
-| STO Modbus adapter crashes                              | ROS system is shutdown which leads to an abrupt stop of the robot. |
-| Connection loss between PNOZmulti/PSS4000 & Modbus client        | Stop 1 is triggered                                     |
-| System overload (messages don't arrive in time)         | In case a Stop 1 message does not arrive in time, the safety controller will automatically perform a hard stop. In case a Stop 1-release message does not get through, brakes will remain closed. |
-| STO Modbus adapter cannot connect to stop services    | ROS system will not start.                              |
-| STO Modbus adapter cannot connect to recover services | Node does start and robot can be moved until a stop is triggered. Afterwards the brakes will remain closed. |
+## Run permitted signal
+The STO function (Safe torque off) of the robot arm is a safety function to immediately turn off torque of the drives. The behavior can be triggered via the
+RUN_PERMITTED signal.
+
```

Comment:
```suggestion

**Please note:**
The system only reacts on changes of the RUN_PERMITTED signal, so it may be necessary to retrigger it in order to enable the drives at start.

```

Reply:
@ct2034 We had a discussion about that today. I thought it makes sense to add it here. What do you think?

Reply:
what do you mean by system? The ROS system? And what should be retriggered?

Reply:
better?